### ISSUE TITLE
Ability to change main character and reorder characters

### ISSUE THREAD

**Issue by divoxx** [datmeal/lostarkchecklist#36]
In the checklist page, it would be great if the order of the characters could be changed, including being able to change who is your main character (maybe by dragging it to the first position?).

Changing main, or changing your alts setup, is currently very cumbersome todo.

Great work on the tool, though. It's great :)

**Comment by datmeal** [comment]
Agreed, more classes are making their way into the western release which means more main changes!
